app.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-classes-per-file */
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+class Books {
+  constructor() {
+    this.books = [];
+  }
+
+  addBook(title, author) {
+    const book = new Book(title, author);
+    const storedBooks = JSON.parse(localStorage.getItem('books'));
+    if (storedBooks == null) {
+      this.books = [];
+    } else {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  removeBook(index) {
+    this.books = JSON.parse(localStorage.getItem('books'));
+    this.books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+}
+
+let books = JSON.parse(localStorage.getItem('books'));
+if (books == null) {
+  books = [];
+}
```

Comment:
- [ ] the intention of this milestone is to use JavaScript classes instead of objects and arrays, please note that you can use objects and arrays but it has to be inside the classes.